imagetest/fixtures.go

```diff
@@ -141,6 +141,56 @@ func (t *TestWorkflow) CreateTestVM(name string) (*TestVM, error) {
 	return &TestVM{name: vmname, testWorkflow: t, instance: i}, nil
 }
 
+// CreateTestVMWithReboot adds the necessary steps to create a VM with the
+// specified name to the workflow, while expecting that it will be rebooted.
+// This should only be used in cases where there is a special interaction
+// with the wrapper binary, such as in the shutdown script tests.
+func (t *TestWorkflow) CreateTestVMWithReboot(name string) (*TestVM, error) {
```

Comment:
I wonder if we can't reuse most of the current `CreateTestVM()` function and only add an argument/flag to make the differentiation.

Reply:
done